fault/tester.py

```diff
@@ -60,7 +60,7 @@ def step(self):
 
     def compile_and_run(self, target="verilator", **kwargs):
         if target == "verilator":
-            target = VerilogTarget(self.circuit, self.test_vectors, **kwargs)
+            target = VerilatorTarget(self.circuit, self.test_vectors, **kwargs)
```

Comment:
why did this change? the class name in the other file seems to still be `VerilatorTarget`

Reply:
this was a typo in the original merge, it should be `VerilatorTarget`, it was `VerilogTarget`

Reply:
👍